From 59ad83b0ff0a67cbc6e194f1175251ba614d937e Mon Sep 17 00:00:00 2001
From: John Audia <therealgraysky@proton.me>
Date: Wed, 4 Jun 2025 09:22:04 -0400
Subject: [PATCH 4/5] wireguard-tools: update to v1.0.20250521

Update to latest version.

Release notes: https://lists.zx2c4.com/pipermail/wireguard/2025-May/008789.html

% wg -v
wireguard-tools v1.0.20250521 - https://git.zx2c4.com/wireguard-tools/

Build system: x86/64
Build-tested: x86/64
Run-tested: x86/64

Signed-off-by: John Audia <therealgraysky@proton.me>
Link: https://github.com/openwrt/openwrt/pull/19023
Signed-off-by: Nick Hainke <vincent@systemli.org>
---
 package/network/utils/wireguard-tools/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/package/network/utils/wireguard-tools/Makefile b/package/network/utils/wireguard-tools/Makefile
index d704577d2c..12127eb12f 100644
--- a/package/network/utils/wireguard-tools/Makefile
+++ b/package/network/utils/wireguard-tools/Makefile
@@ -10,12 +10,12 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=wireguard-tools
 
-PKG_VERSION:=1.0.20210914
-PKG_RELEASE:=4
+PKG_VERSION:=1.0.20250521
+PKG_RELEASE:=1
 
 PKG_SOURCE:=wireguard-tools-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=https://git.zx2c4.com/wireguard-tools/snapshot/
-PKG_HASH:=97ff31489217bb265b7ae850d3d0f335ab07d2652ba1feec88b734bc96bd05ac
+PKG_HASH:=b6f2628b85b1b23cc06517ec9c74f82d52c4cdbd020f3dd2f00c972a1782950e
 
 PKG_LICENSE:=GPL-2.0
 PKG_LICENSE_FILES:=COPYING
-- 
2.43.0

